File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        int num1 = 0;
        int num2 = 1;

        for (int i = 0; i < num_list.length; i++) {
            num1 += num_list[i];
            num2 *= num_list[i];
        }

        int num3 = num1 * num1;

        answer = num2 < num3 ? 1 : 0;

        return answer;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    var num1 = 0;
    var num2 = 1;
     
    for(var i =0; i<num_list.length; i++){
        num1 += num_list[i];
        num2 *= num_list[i];
    }
    return num1**2 > num2 ? 1 : 0;
}